### DISCUSSION TITLE
Barotrauma (overpressure) should gradually, not instantly reduce vitality

### DISCUSSION THREAD

**Discussion by Krzeszny** [FakeFishGames/Barotrauma#5384] [Ideas and suggestions]
Currently, we can sit for almost 3 seconds under extreme pressure until we explode, yet we don't take damage and don't need to recover from it.

My simple suggestion is to **make the barotrauma affliction _gradually_ reduce vitality.**

It would work best if barotrauma was a type of internal damage that can be healed just like any other internal damage.

**Comment by Kulik242** [comment]
Full pressure will kill you instantly as in real life (3 seconds delay cause its still a game). But as water flows into the sub the pressure builds and it may take quite a while to succumb to over-pressure, depends of how big is the breach, i.e. how fast the room pressurizes.

**Reply by Krzeszny** [reply]
I still think not having to recover from being exposed to overpressure is weird.
